File: NetNodePluginServer/NetNodeClient/Request/ServerNodeRequest.py
from EtherealC.Request.Decorator.Request import Request
from EtherealC.Request.WebSocket.WebSocketRequest import WebSocketRequest
from EtherealS.Net.NetNode.Model.NetNode import NetNode


class ServerNodeRequest(WebSocketRequest):
    def __init__(self):
        super().__init__()
        self.name = "ServerNetNodeService"
        self.types.add(type=int, type_name="Int")
        self.types.add(type=str, type_name="String")
        self.types.add(type=bool, type_name="Bool")
        self.types.add(type=type(NetNode()), type_name="NetNode")


    @Request()
    def Register(self, node: NetNode) -> bool:
        pass

File: NetNodePluginServer/NetNodeClient/Request/ServerNodeRequest.cs
using NetNodePlugin.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NetNodePlugin.NetNodeClient.Request
{
    public class ServerNodeRequest : EtherealC.Request.WebSocket.WebSocketRequest,IServerNodeRequest
    {
        public ServerNodeRequest()
        {
            name = "ServerNetNodeService";
            types.Add<int>("Int");
            types.Add<long>("Long");
            types.Add<string>("String");
            types.Add<bool>("Bool");
            types.Add<Model.NetNode>("NetNode");
        }

        public override void Initializate()
        {

        }

        public bool Register(NetNode node)
        {
            return true;
        }

        public override void UnInitialize()
        {

        }
    }
}
